models/ios/evpn_global/ios_evpn_global.yaml

```diff
@@ -0,0 +1,89 @@
+module: ios_evpn_global
+short_description: Resource module to configure L2VPN EVPN.
+description: This module provides declarative management of L2VPN EVPN on Cisco IOS network
+  devices.
+version_added: 1.0.0
+author: Padmini Priyadarshini Sivaraj (@PadminiSivaraj)
+notes:
+  - Tested against Cisco IOSl2 device with Version 15.2 on VIRL.
+  - Starting from v2.5.0, this module will fail when run against Cisco IOS devices that do
```

Comment:
Is this statement verified? Does this stand correct for the specific module?

Reply:
Removed this statement.